setup.py

```diff
@@ -23,23 +23,18 @@ def find_version(*file_paths):
     raise RuntimeError("Unable to find version string.")
 
 
-requires = ['jmespath>=0.7.1,<1.0.0',
-            'python-dateutil>=2.1,<2.7.0',
-            'docutils>=0.10']
-
-
-if sys.version_info[:2] == (2, 6):
-    # For python2.6 we have a few other dependencies.
-    # First we need an ordered dictionary so we use the
-    # 2.6 backport.
-    requires.append('ordereddict==1.1')
-    # Then we need simplejson.  This is because we need
-    # a json version that allows us to specify we want to
-    # use an ordereddict instead of a normal dict for the
-    # JSON objects.  The 2.7 json module has this.  For 2.6
-    # we need simplejson.
-    requires.append('simplejson==3.3.0')
-
+requires = [ 'setuptools>=21.0',
```

Comment:
This actually only needs to go in `requirements.txt` I believe. By the time you pass it to `install_requires` it's too late (since the old version of `setuptools` is already imported). I think the same goes for `setup.cfg`.

After PEP 518, this will go in `pyproject.toml`.